python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -246,6 +246,32 @@ def __exit__(self, *exc) -> None:
         self._exit_stack.close()
         self._logger = None
 
+    def merge_resources(self, resources_dict: Mapping[str, Any]) -> "RunStatusSensorContext":
```

Comment:
where does this get invoked? is this override of a parent class impl or something?

Reply:
it's called in `get_or_create_sensor_context`, which we start calling here: https://github.com/dagster-io/dagster/pull/17788/files#diff-32ef053cc1ce6eaf22c65bbaf654c262324ec74bf6e8eb2b037da899f9793a62L832

this merges resources with the provided context, if any are provided 'loose' (as kwargs)